Ontology/Information/TagSet/CapabilityTagSet.json

```diff
@@ -0,0 +1,205 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:device:CapabilityTagSet;1",
+  "@type": "Interface",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "capabilityAssetComponent",
```

Comment:
Do we need to have the prefix "capability" before each Property given that these are all part of the "tags" Component on the Capability model?

Reply:
I would say No